notebooks/utils/download/parsertools/jsonparser.py

```diff
@@ -0,0 +1,36 @@
+import json
+
+
+class JSONParser:
+    def __init__(self) -> None:
+        pass
+
+    def parse(self, json_file):
+        with open(json_file) as file:
+            raw_metadata = json.load(file)
+
+        new_metadata = {
+            "id": raw_metadata["DC.Identifier"],
+            "title": raw_metadata["citation_title"],
+            "lang": raw_metadata["citation_language"],
+            "author": raw_metadata["citation_author"],
+            "date": raw_metadata["citation_date"],
+            "type": raw_metadata["DC.Type.articleType"],
+        }
+
+        try:
+            new_metadata["vol"] = raw_metadata["citation_volume"]
+        except KeyError:
+            new_metadata["vol"] = "Undefined"
```

Comment:
When trying to get a key from a dict (with the possibility of the key not being in it) there are a couple of ways to do it more Pythonically! A couple:

```suggestion
new_metadata["vol"] = raw_metadata.get("citation_volume", "undefined")
```

Reply:
This replaces the entire `try-except` block.